ci/sign-all-tasks.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  layoutRoot: ''
+
+steps:
+# Guard against missing parameters
+- ${{ if eq(parameters.layoutRoot, '') }}:
+  - |
+      Badly configured build; an expected parameter is missing. //
+      layoutRoot: ${{ parameters.layoutRoot }}
+
+- task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+  inputs:
+    ConnectedServiceName: AzurePipelinesTasksESRP
+    FolderPath: '${{ parameters.layoutRoot }}\_package\nested-zips-layout'
+    Pattern: '*.zip'
+    signConfigType: inlineSignParams
+    inlineOperation: |
+      [{
+      "keyCode": "CP-401405",
+      "operationSetCode": "NuGetSign",
+      "parameters": [],
+      "toolName": "sign",
+      "toolVersion": "1.0"
+      },
+      {
+      "keyCode": "CP-401405",
+      "operationSetCode": "NuGetVerify",
+      "parameters": [],
+      "toolName": "sign",
+      "toolVersion": "1.0"
+      }
+      ]
+  displayName: Sign Task Nuget Packages
+  condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'), ne(variables['numTasks'], 0)) 
```

Comment:
Maybe this condition should go in the parent so it's more clear when this happens?

Reply:
Template inclusion can't have conditions :(